internal/cli/up.go

```diff
@@ -13,9 +17,10 @@ type upCmd struct{}
 
 func (c *upCmd) register() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "up <servicename>",
-		Short: "stand up a service",
-		Args:  cobra.ExactArgs(1),
+		Use:           "up <servicename>",
+		Short:         "stand up a service",
+		Args:          cobra.ExactArgs(1),
+		SilenceErrors: true,
```

Comment:
hmmm...it seems weird to use RunE (which means "handle errors upstream") and at the same use SilenceErrors (which means "don't handle errors upstream")

maybe we could switch the RunE to a Run, and do the error handling ourselves, like Pat did in the pets code?